okhttp/src/jvmMain/kotlin/okhttp3/internal/publicsuffix/PublicSuffixDatabase.kt

```diff
@@ -112,7 +111,7 @@ class PublicSuffixDatabase {
     }
 
     // Break apart the domain into UTF-8 labels, i.e. foo.bar.com turns into [foo, bar, com].
-    val domainLabelsUtf8Bytes = Array(domainLabels.size) { i -> domainLabels[i].toByteArray(UTF_8) }
+    val domainLabelsUtf8Bytes = Array(domainLabels.size) { i -> domainLabels[i].toByteArray() }
```

Comment:
This is a behavior change. If you want to remove parameters, please do that in a separate PR. This breaks our behavior on any JVM that doesn’t have UTF-8 as a default charset.

Reply:
Reverted.